carddrawer/src/test/java/com/meli/android/carddrawer/configuration/AccountMoneyDefaultConfigurationTest.kt

```diff
@@ -0,0 +1,88 @@
+package com.meli.android.carddrawer.configuration
+
+import com.meli.android.carddrawer.R
+import com.meli.android.carddrawer.model.CardAnimationType
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.MockitoAnnotations
+import org.mockito.junit.MockitoJUnitRunner
+
+@RunWith(MockitoJUnitRunner::class)
+class AccountMoneyDefaultConfigurationTest {
+
+    private lateinit var accountMoneyDefaultConfiguration: AccountMoneyDefaultConfiguration
+
+    @Before
+    fun init() {
+        MockitoAnnotations.initMocks(this)
+        accountMoneyDefaultConfiguration = AccountMoneyDefaultConfiguration()
+    }
+
+    @Test
+    fun `should return BankImageRes`() {
```

Comment:
Guys, we were using something along the lines of "Given - when - then" for the tests names.

I think at least the when part should be there.

So, this test name should be "when getting account money bank res then it should return 0"

Reply:
Test names have been changed according to this pattern